pkg/storage/fs/owncloud/owncloud.go

```diff
@@ -881,6 +960,7 @@ func (fs *ocfs) ListGrants(ctx context.Context, ref *provider.Reference) (grants
 	grants = make([]*provider.Grant, 0, len(aces))
 	for i := range aces {
 		grantee := &provider.Grantee{
+			// TODO lookup uid from principal
```

Comment:
- [ ] The user lookup here needs to be implemented before the sharing tests can pass. at least I think they should fail somewhere without it.